examples/terraform/devenv.yaml

```diff
@@ -0,0 +1,6 @@
+allowUnfree: true
+inputs:
+  nixpkgs:
```

Comment:
Let's leave out nixpkgs, so that the defaults apply

Reply:
Done!